src/render/src/encoder.rs

```diff
@@ -39,11 +111,11 @@ pub enum CopyError<S, D> {
 /// Result type returned when copying a buffer into another buffer.
 pub type CopyBufferResult = Result<(), CopyError<usize, usize>>;
 
-/// Result type returned when copying buffer data into a texture.
-pub type CopyBufferTextureResult = Result<(), CopyError<usize, [texture::Size; 3]>>;
+/// Result type returned when copying buffer data into an image.
+pub type CopyBufferImageResult = Result<(), CopyError<usize, [image::Size; 3]>>;
```

Comment:
btw, could use `image::Dimensions`?

Reply:
It can be both dimensions or an offset, so I don't think that `AaMode` makes sense here.

Reply:
ok